containers/src/CMakeLists.txt

```diff
@@ -28,6 +28,9 @@ KOKKOS_LIB_INCLUDE_DIRECTORIES(kokkoscontainers
   ${CMAKE_CURRENT_BINARY_DIR}
   ${CMAKE_CURRENT_SOURCE_DIR}
 )
+
+KOKKOS_LINK_TPL(kokkoscontainers PUBLIC ROCTHRUST)
```

Comment:
Why link containers?

Reply:
My bad, I meant to add this in algorithm.